Dockerfile

```diff
@@ -15,13 +15,16 @@ RUN groupadd -r spark && useradd -u 1001 -r -g spark spark_user
 RUN apt-get update && apt-get install -y \
     # tools for troubleshooting network issues
     iputils-ping dnsutils netcat-openbsd \
+    # Redis client for interacting with Redis server
+    redis-tools \
```

Comment:
I wouldn't expect to run redis tools inside the spark container, but this doesn't really hurt either